<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>145</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>185</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/5.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>65</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/3.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>25</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/4.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>105</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/1.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>640</width>
     <height>260</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{

background-image: url(:/new/prefix/groupbox3.jpg);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(226, 25, 25);
font: 24pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>监测界面</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>240</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>160</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>勒克斯</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix/3.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>155</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix/5.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>85</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefi/1.jpg);
image: url(:/new/prefix/1.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>125</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>当前光照</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>当前气压</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>当前湿度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>55</y>
       <width>45</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>摄氏度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>25</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>当前转速</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>120</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix/2.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>15</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix/4.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>25</y>
       <width>28</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>m/h</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>125</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>Pa</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>55</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>当前温度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>32</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>370</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox_Month">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>50</y>
       <width>75</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>请选择日期</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>曲线显示</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>20</y>
       <width>120</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>请选择喷灌机编号</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>距喷灌完成还需要</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>50</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>月</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Year">
     <property name="geometry">
      <rect>
       <x>92</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>2016</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2017</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2018</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>h</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_bianhao">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>喷灌机已运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>h</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Day">
     <property name="geometry">
      <rect>
       <x>252</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>01</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>02</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>03</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>04</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>05</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>06</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>07</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>08</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>09</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>50</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>年</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>303</x>
       <y>50</y>
       <width>16</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>日</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>查询数据库</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;楷体&quot;;</string>
     </property>
     <property name="text">
      <string>wendu</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
